### DISCUSSION TITLE
Android: Some emojis not colored anymore

### DISCUSSION THREAD

**Discussion by htcfreek** [community/community#4159] [Mobile] [answered]
After the last Update of Github Mobile for Android the emojis aren't colored anymore. This is happening in issue and PR-comments.

Example: ❤️

App version after update:
Android: 11
Device: Samsung Galaxy A41

**Comment by htcfreek** [comment]
App version after update: 1.18.0

**Comment by alcere** [comment]
Thanks for the feedback! I've created an issue to look at this and fix it as soon as possible!

**Comment by htcfreek** [comment, marked as answer]
@alcere 
It seems to be fixed.
